pages/extensions-Phonos.xml

```diff
@@ -45,87 +45,87 @@
 |-
-| key || {{#phonos: text=key | ipa=k̟ʰi | lang=en-US }}
+| key || {{#phonos: text=key | ipa=/k̟ʰi/ | lang=en }}
 |-
-| vin blanc || {{#phonos: text=vin blanc | ipa=vɛ̃ blɑ̃ | lang=fr-FR }}
+| vin blanc || {{#phonos: text=vin blanc | ipa=/vɛ̃ blɑ̃/ | lang=fr }}
 |-
-| wean || {{#phonos: text=wean | ipa=ˈwɪən | lang=sco }}
+| wean || {{#phonos: text=wean | ipa=/ˈwɪən/ | lang=sco }}
 |-
-| llandudno || {{#phonos: text=llandudno | ipa=ɬanˈdɨdnoː | lang=cy-GB }}
+| llandudno || {{#phonos: text=llandudno | ipa=/ɬanˈdɨdnoː/ | lang=cy }}
 |-
-| नमस्ते || {{#phonos: text=नमस्ते | ipa=nə.məs.t̪eː | lang=hi-IN }}
+| नमस्ते || {{#phonos: text=नमस्ते | ipa=/nə.məs.t̪eː/ | lang=hi }}
 |-
-| நன்றி || {{#phonos: text=நன்றி | ipa=n̪anri | lang=ta }}
+| நன்றி || {{#phonos: text=நன்றி | ipa=/n̪anri/ | lang=ta }}
 |-
-| Montréal || {{#phonos: text=Montréal | ipa=mɔ̃.ʁe.al | lang=fr-FR }}
+| Montréal || {{#phonos: text=Montréal | ipa=/mɔ̃.ʁe.al/ | lang=fr }}
 |-
-| Louisville || {{#phonos: text=Louisville | ipa=ˈlʊvɪl | lang=en-US }}
+| Louisville || {{#phonos: text=Louisville | ipa=/ˈlʊvɪl/ | lang=en }}
 |-
-| Shibboleth || {{#phonos: text=Shibboleth | ipa=ˈʃɪbəlɛθ | lang=en-US }}
+| Shibboleth || {{#phonos: text=Shibboleth | ipa=/ˈʃɪbəlɛθ/ | lang=en }}
 |-
-| ευχαριστώ || {{#phonos: text=ευχαριστώ | ipa=ef.xa.ɾiˈsto | lang=el }}
+| ευχαριστώ || {{#phonos: text=ευχαριστώ | ipa=/ef.xa.ɾiˈsto/ | lang=el }}
 |-
-| chocolate || {{#phonos: text=chocolate | ipa=ˈt͡ʃɔk(ə)lɪt | lang=en-US }}
+| chocolate || {{#phonos: text=chocolate | ipa=/ˈt͡ʃɔk(ə)lɪt/ | lang=en }}
 |-
-| sushi || {{#phonos: text=sushi | ipa=ˈsʊʃi | lang=en-US }}
+| sushi || {{#phonos: text=sushi | ipa=/ˈsʊʃi/ | lang=en }}
 |-
-| спасибо || {{#phonos: text=спасибо | ipa=spɐˈsʲibə | lang=ru-RU }}
+| спасибо || {{#phonos: text=спасибо | ipa=/spɐˈsʲibə/ | lang=ru }}
 |-
-| supercalifragilisticexpialidocious || {{#phonos: text=supercalifragilisticexpialidocious | ipa=suːpəˌkælɨˌfɹædʒɨˌlɪstɪkˌɛkspɪˌælɨˈdəʊʃəs | lang=en-US }}
+| supercalifragilisticexpialidocious || {{#phonos: text=supercalifragilisticexpialidocious | ipa=/suːpəˌkælɨˌfɹædʒɨˌlɪstɪkˌɛkspɪˌælɨˈdəʊʃəs/ | lang=en }}
 |-
-| Cornwall || {{#phonos: text=Cornwall | ipa=ˈkɔːrnwɔːl | lang=en-US }}
+| Cornwall || {{#phonos: text=Cornwall | ipa=/ˈkɔːrnwɔːl/ | lang=en }}
 |-
-| Thames || {{#phonos: text=Thames | ipa=tɛmz | lang=en-US }}
+| Thames || {{#phonos: text=Thames | ipa=/tɛmz/ | lang=en }}
 |-
-| Gunwalloe || {{#phonos: text=Gunwalloe | ipa=gʌnˈwɒləʊ | lang=en-US }}
+| Gunwalloe || {{#phonos: text=Gunwalloe | ipa=/gʌnˈwɒləʊ/ | lang=en }}
 |-
-| Porthleven || {{#phonos: text=Porthleven | ipa=ˌpɔːrθˈlɛvən | lang=en-US }}
+| Porthleven || {{#phonos: text=Porthleven | ipa=/ˌpɔːrθˈlɛvən/ | lang=en }}
 |-
-| Somerset || {{#phonos: text=Somerset | ipa=ˈsʌmərsɛt | lang=en-US }}
+| Somerset || {{#phonos: text=Somerset | ipa=/ˈsʌmərsɛt/ | lang=en }}
 |}</text>
       <sha1>60lcj0j0xism45kiuo959zs7xx2xywl</sha1>
```

Comment:
Funny, MediaWiki doesn't care about this hash being wrong, there's not even a warning.

Reply:
My preferred technique is to edit the page, then re-export. We probably should care in case this ever enforced in the future.

Reply:
Deleting it would also be fine, all of the fields in the dumps seem to be optional.